src/InvokePlaster.ps1

```diff
@@ -1512,3 +1514,14 @@ function GetPlasterManifestPathForCulture([string]$TemplatePath, [ValidateNotNul
 
     $null
 }
+
+
+# Dot source the individual module command scripts.
+. .\NewPlasterManifest.ps1
+. .\TestPlasterManifest.ps1
+. .\GetPlasterTemplate.ps1
+# . .\InvokePlaster.ps1
+. .\WritePlasterParameter.ps1
+. .\WritePlasterManifestContent.ps1
+
+Invoke-Plaster c:\GitWork\Clones\GitHub\�Tom\ModuleBuild\release\ModuleBuild\plaster\ModuleBuild\ c:\Temp\--\-PSModul\ModuleBuildTest2\
```

Comment:
I'm not sure what is up with this but it needs to be removed.

Reply:
I'm very sorry that I missed this debugging code and checked it in.
Thanks for removing it!